### ISSUE TITLE
Image gallery listing variation: Disable play-pause button for screen-readers

### ISSUE THREAD

**Issue by jackahl** [plone/volto#6303]
When navigating the image gallery listing template with a screenreader, you can start the slideshow using the respecive button. But this then causes the screen-reader to "freak out" because it will try to read out the current images alt tag, but cant finish that before the next image comes up. I dont think the whole play function is not really suited to screen-readers. If you have an opinion on this feel free to share! I am open to suggestions, on how else to handle this.

**Comment by davisagli** [comment]
I agree, I would look for a way to hide this button from screenreaders and make sure they have good controls for reading through the slides. (I am not sure if it is better to use the Next/Prev buttons for this or just read the slides in a sequence as though they were in a normal list.)
